docs/components/async.md

```diff
@@ -20,6 +22,36 @@ async rerender_on: 'my_event' do
 end
 ```
 
+**Attention:** The `rerender_on` option lets you rerender parts of your UI asynchronously, which is cool. It does come with an implications that could lead to unintended behaviour, though. Take a look at the code below:
+
+```ruby
+class Pages::ExamplePage < Page::Cell::Page
+
+  def prepare
+    user = User.last
+  end
+
+  # ...
+
+  def response
+    components {
+      async rerender_on: 'my_event' do
+        div id: 'my-div' do
+          plain user.name
+        end
+      end
+    }
+  end
+
+end
+```
+
+Firstly, the async component gets displayed on initial pageload, showing the most recently added user's name. On every occurance of `my_event`, the `prepare` method gets called, again fetching the most recently added user from the DB. This could lead to 1) unwanted information on the UI adn 2) a lot of unnecessary DB queries. We recommend to keep a close eye on the async component and, for this example, calling a partial with the DB query within the `div id: 'my-div'`.
```

Comment:
1) You dramatize this behaviour. **Rerendering** is supposed to fetch the latest state from the server... I don't know how this could lead to unintentionally showing unwanted information on the UI. 

2) We have to clarify the timing of DB queries according to their position on the UI. I think, currently they are always executed, even if they are exclusively defined in a block, which is not rerendered for example. I will take care of this when implementing **caching**

Reply:
I hope it gets clearer with the update I just pushed :)